torch/utils/tensorboard/writer.py

```diff
@@ -579,7 +579,7 @@ def add_image_with_boxes(self, tag, img_tensor, box_tensor, global_step=None,
             dataformats (string): Image data format specification of the form
               NCHW, NHWC, CHW, HWC, HW, WH, etc.
         Shape:
-            img_tensor: Default is :math:`(3, H, W)`. It can be specified with ``dataformat`` agrument.
+            img_tensor: Default is :math:`(3, H, W)`. It can be specified with ``dataformats`` argument.
```

Comment:
nit: with the ```:attr:`dataformats` ``` argument

Reply:
@vishwakftw  `:attr` shouldn't be added because it is used for denoting class attributes. See #25885

Reply:
I thought that for functions, the arguments could use :attr: in the doc strings

Reply:
But I see that this is a function in a class, please regard my comment.

Reply:
Im sorry, I don't understand, should I change \`\`dataformats\`\` argument to :attr:\`dataformats\` argument?

Reply:
ops, just read the last couple comments, my bad.